tests/unit/test_misc.py

```diff
@@ -173,13 +173,14 @@ def test_add_waveforms(self):
                     [1, 2, 3]   # spike 4
                 ]
             ]
-        ut.add_unit(waveforms=wf1)
+        ut.add_unit(waveforms=wf1, unit='volts', conversion=1., offset=0.)
```

Comment:
@rly I was wondering if it would be easiest to just have these specified either (a) the first time waveforms are passed in `add_unit`, (b) every time they are passed in `add_unit`, or (c) they should be defined on the first call of `Units(...)` with defaults set in case no waveforms are intended to be added to the table (which is also kind of what I was trying to do above).

Reply:
It depends on what use case we want to support -- a single conversion and offset for all units or a conversion and offset for each unit. In my experience, the latter case, where there is more than one set of conversion and offset for all the units is much less common, so I think we need not support it by default. In that case, we could do something like what has been implemented for waveform unit and rate, where it is set on the constructor